File: app/src/androidTest/java/ie/noel/dunsceal/persistence/db/TestData.java
package ie.noel.dunsceal.persistence.db;

import java.util.Arrays;
import java.util.Date;
import java.util.List;

import ie.noel.dunsceal.models.entity.DunEntity;
import ie.noel.dunsceal.models.entity.EntranceEntity;
import ie.noel.dunsceal.models.entity.InvestigationEntity;

/**
 * Utility class that holds values to be used for testing.
 */
public class TestData {

    static final DunEntity DUN_ENTITY = new DunEntity(1, "name", "desc", 3);
    static final DunEntity DUN_ENTITY2 = new DunEntity(2, "name2", "desc2", 20);

    static final List<DunEntity> DUNS = Arrays.asList(DUN_ENTITY, DUN_ENTITY2);

    static final InvestigationEntity INVESTIGATION_ENTITY = new InvestigationEntity(1, DUN_ENTITY.getId(),
            "desc", new Date());
    static final InvestigationEntity INVESTIGATION_ENTITY2 = new InvestigationEntity(2,
            DUN_ENTITY2.getId(), "desc2", new Date());

    static final List<InvestigationEntity> INVESTIGATIONS = Arrays.asList(INVESTIGATION_ENTITY, INVESTIGATION_ENTITY2);

    static final EntranceEntity ENTRANCE_ENTITY = new EntranceEntity(1, DUN_ENTITY.getId(),
        "desc", new Date());
    static final EntranceEntity ENTRANCE_ENTITY2 = new EntranceEntity(2,
        DUN_ENTITY2.getId(), "desc2", new Date());

    static final List<EntranceEntity> ENTRANCES = Arrays.asList(ENTRANCE_ENTITY, ENTRANCE_ENTITY2);
}

File: app/src/androidTest/java/ie/noel/dunsceal/persistence/db/TestData.kt
package ie.noel.dunsceal.persistence.db

import ie.noel.dunsceal.models.entity.Dun
import ie.noel.dunsceal.models.entity.InvestigationEntity
import java.util.*

/**
 * Utility class that holds values to be used for testing.
 */
object TestData {
  val DUN_ENTITY = Dun(1, "name", "desc", 3)
  val DUN_ENTITY2 = Dun(2, "name2", "desc2", 20)
  val DUNS = Arrays.asList(DUN_ENTITY, DUN_ENTITY2)
  val INVESTIGATION_ENTITY = InvestigationEntity(1, DUN_ENTITY.id,
      "desc", Date())
  val INVESTIGATION_ENTITY2 = InvestigationEntity(2,
      DUN_ENTITY2.id, "desc2", Date())
  val INVESTIGATIONS = Arrays.asList(INVESTIGATION_ENTITY, INVESTIGATION_ENTITY2)
}